### DISCUSSION TITLE
when installing I have a problem with nic cards networks cards

### DISCUSSION THREAD

**Discussion by codefaztaz** [Security-Onion-Solutions/securityonion#9972] [General]
when I am installing security onion when I have to select the networks cards I can not select any of them.

I dont know how to fix this issue.

I am installing security onion 2 last version.

**Comment by cm-ops** [comment]
Can you provide a little more information? What are you installing on (VM or bare metal) at a minimum.

**Comment by codefaztaz** [comment]
I am installing not in a VM. I am installing on a computer. in the step select the interface I can not select.

securityonion-2.3.220-20230301.iso this is the version.

**Comment by codefaztaz** [comment]
mi both nic cards are intel 80.....

**Comment by cm-ops** [comment]
Do you see the NIC in the list in the setup screen?

**Comment by codefaztaz** [comment]
yes I see the 2 nic in the list. but I can not select any of them

**Comment by cm-ops** [comment]
To confirm, if you hit the spacebar to select one, nothing happens?

**Comment by vanh20** [comment]
Compare the checksum of your downloaded iso. I can't find the original post now as new version is already released. Mine is 76870cf09ff27893574fc104f9ac6642 (md5sum). It checked fine when I downloaded it, though I'm not sure it is still the same as the value now. I did not save the original value.
